src/pages/Facility/settings/locations/components/LocationTable.tsx

```diff
@@ -264,7 +268,16 @@ export function LocationTable({
                         {/* Delete button or spacer */}
                         {!location.has_children &&
                         !location.current_encounter ? (
-                          <AlertDialog>
+                          <AlertDialog
```

Comment:
I'd suggest following a similar pattern to edit button instead, add a button inside the tooltip, use it to update state and use that state to trigger alert dialog.